### ISSUE TITLE
Digital Out doesn't seem to working

### ISSUE THREAD

**Issue by kotha16** [osy86/HaC-Mini#189]
The Digital Out on the back of the Hades Canyon doesn't seem to be outputting sound. The headphone output on the front works though.

**Comment by osy86** [comment]
Did you select digital out in the sound options?

**Comment by kotha16** [comment]
Yes i did. No sound output from the digital

**Comment by osy86** [comment]
Did it work on 2.5? https://github.com/osy86/HaC-Mini/releases/tag/v2.5

**Comment by chrisallen** [comment]
If the switch from `Line Out` to `Speaker` for the rear port causes this issue, then Switching it back to the old behaviour is fine. @osy86 would you be able to have the front port available at the same time as the rear line out? that way I can switch output in the menu bar to `Headphones` manually when they are plugged in?

**Comment by kotha16** [comment]
> Did it work on 2.5? https://github.com/osy86/HaC-Mini/releases/tag/v2.5

No

**Comment by osy86** [comment]
1) make sure you don't have something plugged in to both back and front port. 2) make sure it works in windows/linux.

**Comment by kotha16** [comment]
It worked in windows and I just plugged it in to one

**Comment by osy86** [comment]
Can you also try v2.4? Maybe a recent applealc change broke things. https://github.com/osy86/HaC-Mini/releases/tag/v2.4

**Comment by edwardwang0302** [comment]
Any updates? Every time when I reboot from windows, sound digital output doesn't work. I have already updated to the lastest version 2.9.
And My temporary solution is shutdown, reset nvram. Any suggestion?

**Comment by osy86** [comment]
Does it work from a fresh boot? There's quite a few issues resulting from rebooting from windows due to driver incompatibilities.

**Comment by edwardwang0302** [comment]
> Does it work from a fresh boot? There's quite a few issues resulting from rebooting from windows due to driver incompatibilities.

Thanks for your answer. I tested it just now. A fresh boot is ok. But it can't work if I reboot from windows directly.

**Comment by osy86** [comment]
I don’t think this is limited to digital out. I think normal output sometimes doesn’t work either. It’s an incapability with AppleALC and is unlikely to be fixed. I think Clover had an option to reset HDA but I don’t know if it’s possible with OC. Personally I don’t think it’s worth fixing but if anyone else has an idea, please let us know.

**Comment by i4phone** [comment]
i find the same issue ,no audio out (digital port) after restart from windows . but if restart from mac or just power the machine ,the audio out is ok ,but only for the rear port.  the Front audio out is alway noise.

**Comment by unsungloser** [comment]
can something like this work?
https://www.tonymacx86.com/threads/solved-no-audio-after-reboot-from-windows-applehda-w-alc-668.187624/

**Comment by osy** [comment]
You can try it and report back.
